core/server/data/schema/schema.js

```diff
@@ -333,6 +333,21 @@ module.exports = {
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
     },
+    labels: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        name: {type: 'string', maxlength: 191, nullable: false, unique: true},
```

Comment:
What's the reason for the `unique` here? Wast thinking the model would be more like `tags` :thinking:

Reply:
There was a slight change in that pattern for `labels` after discussion with Zimo, where we now want `name` to be unique as well in case of a label, as thats the common pattern used across different platforms we looked at. So this diverged from `tags` model and is also why needed a few extra checks in models/serializer to ensure we don't save duplicate label names as well as case-insensitive versions of it

Reply:
roger, makes sense :+1: